microcosm_secretsmanager/loaders/conventions.py

```diff
@@ -0,0 +1,61 @@
+"""
+Load configuration according to conventions:
+
+Configuration is loaded from secretsmanager using the following rules:
+
+ 1. Loads the JSON file from `secrests/{env}/{service_name}`
+
+ 2. Use the version in MICROCOSM_CONFIG_VERSION
+
+If the microcosm environment variables are not present, it skips loading secretsmanager
+
+
+"""
+from os import environ
+
+from microcosm.loaders import load_from_dict
+
+from microcosm_secretsmanager.loaders.base import SecretsManagerLoader
+
+
+def load_from_secretmanager(environment=None):
```

Comment:
If you did
```
def load_from_secretmanager(environment=environ):
```

Your code would be simpler, and the calling convention would be straight-forward